src/rbac/application/project/repository/project_member_repository_impl.py

```
@@ -0,0 +1,74 @@
+                self.db.add(member)
+                saved_members.append(member)
+
+        self.db.commit()
+
+        # 객체 동기화
+        for member in saved_members:
+            self.db.refresh(member)
+        return saved_members
+
+    def update_all(self, members: List[ProjectMember]) -> List[ProjectMember]:
+        updated_members = []
```

Ruff (0.8.2)

Local variable `updated_members` is assigned to but never used

Remove assignment to unused variable `updated_members`

(F841)